sql/alps_submitter_test.go

```diff
@@ -64,13 +66,7 @@ func TestColumnResolve(t *testing.T) {
 	a.Equal(10, nl2.Shape)
 	a.Equal([]int{1, 10}, bl.Boundaries)
 
-	a.Equal("NumericColumn", getFeatureColumnType(cl.Keys[1]))
```

Comment:
why removing the following assertion?

Reply:
That is a previous mistake. All keys of `cross` must be either string or categorical column except HashedCategoricalColumn, given `NumericColumn` which is not supported.